tests/tasks/test_task.py

```diff
@@ -0,0 +1,91 @@
+import sys
+import types
+
+if sys.version_info[0] >= 3:
+    from unittest import mock
+else:
+    import mock
+
+import numpy as np
+
+import openml
+from openml.testing import TestBase
+
+
+class OpenMLTaskTest(TestBase):
+
+    def test_get_clustering_task(self):
+        self.assertRaisesRegexp(KeyError, 'oml:target_feature',
+                                openml.tasks.get_task, 10128)
+
+    @mock.patch('openml.datasets.get_dataset')
+    def test_get_dataset(self, patch):
+        patch.return_value = mock.MagicMock()
+        mm = mock.MagicMock()
+        patch.return_value._retrieve_class_labels = mm
+        patch.return_value._retrieve_class_labels.return_value = 'LA'
+        retval = openml.tasks.get_task(1)
+        self.assertEqual(patch.call_count, 1)
+        self.assertIsInstance(retval, openml.OpenMLTask)
+        self.assertEqual(retval.class_labels, 'LA')
+
+    def test_get_X_and_Y(self):
```

Comment:
hmm if we want to follow sklearn api, ``Y`` should be lowercase (because it is usually a vector). Not sure if that was my fault.